src/s2s/codec/config.cpp

```
@@ -0,0 +1,37 @@
+    c.hop_length = static_cast<int>(loader.get_u32("eartts_codec.hop_length"));
+    c.sample_rate = static_cast<int>(loader.get_u32("eartts_codec.sample_rate"));
+    c.base_hidden_size = static_cast<int>(loader.get_u32("eartts_codec.base_hidden_size"));
+    c.latent_dim = static_cast<int>(loader.get_u32("eartts_codec.latent_size"));
+    c.conv_next_kernel = static_cast<int>(loader.get_u32("eartts_codec.conv_next_kernel"));
+    c.intermediate_ratio = static_cast<int>(loader.get_u32("eartts_codec.intermediate_ratio"));
+    c.num_blocks_per_stage = static_cast<int>(loader.get_u32("eartts_codec.num_blocks_per_stage"));
+    c.n_quantizers = static_cast<int>(loader.get_u32("eartts_codec.num_quantizers"));
+    c.codebook_size = static_cast<int>(loader.get_u32("eartts_codec.codebook_size"));
+    c.stage_channels = loader.get_i32_array("eartts_codec.stage_channels");
+    c.rates = loader.get_i32_array("eartts_codec.rates");
+    if (c.n_quantizers == 0 || c.latent_dim == 0 || c.rates.empty())
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)